src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheck.java

```diff
@@ -96,6 +96,7 @@
  * Property {@code excludeClassesRegexps} - Specify user-configured regular
  * expressions to ignore classes.
  * Type is {@code java.lang.String[]}.
+ * Validation type is {@code Pattern}.
```

Comment:
Please use fully qualified name for type, I hope we did it for other places.

Reply:
Done